hazelcast-spring/src/main/resources/hazelcast-spring-3.6.xsd

```diff
@@ -92,7 +92,7 @@
                                         </xs:documentation>
                                     </xs:annotation>
                                 </xs:attribute>
-                                <xs:attribute name="backup-count" use="optional" type="xs:string">
+                                <xs:attribute name="backup-count" use="optional" type="parameterized-backup-count">
```

Comment:
@mesutcelik, because this value can be integer or parameterized string `${value}`.. it shouldn't be string.